libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -879,3 +882,72 @@ string YulUtilFunctions::suffixedVariableNameList(string const& _baseName, size_
 	}
 	return result;
 }
+
+string YulUtilFunctions::conversionFunctionSpecial(Type const& _from, Type const& _to)
+{
+	string functionName =
+		"convert_" +
+		_from.identifier() +
+		"_to_" +
+		_to.identifier();
+	return m_functionCollector->createFunction(functionName, [&]() {
+		if (_from.category() == Type::Category::StringLiteral)
```

Comment:
I suggest to revert the `if` condition so the rest doesn't need an extra indent

Reply:
This might change in the future, but I'll do it for now.